stratum/hal/lib/common/testdata/invalid_iface_component.pb.txt

```diff
@@ -38,8 +38,5 @@ interface {
     id {
       value: 1
     }
-    ifindex {
-      value: 1
-    }
```

Comment:
Can someone confirm that this removal is still needed after the latest design change?
My understanding of the test code is very minimal.

Reply:
Yes, I believe it is still necessary. The openconfig_converter translates between oc.Device and ChassisConfig, and thus only parts of the tree that are present in both should be included. More practically, without a pointer to switch interface, we can't fill in the ifindex part anymore, and it is also part of the "state" tree, instead of the "config" tree. In this case, "id" corresponds to the SDN port number (or ChassisConfig SingletonPort ID), and we shouldn't be validating or including the ifindex as part of the config.